### PR TITLE
Updated code quality ignored bare except & unicode function warnings changes in namecoin.py

### PR DESCRIPTION
**Pull request by kdcis**
Added ignore comments for bare except & unicode function warnings

### CODE REVIEW

File: src/namecoin.py [hunk 1]
```diff
@@ -70,31 +70,31 @@ def __init__(self, options=None):
...
             elif self.nmctype == "nmcontrol":
-                res = self.callRPC("data", ["getValue", string])
+                res = self.callRPC("data", ["getValue", identity])
                 res = res["reply"]
                 if not res:
-                    return (tr._translate(
+                    return (_translate(
                         "MainWindow", 'The name %1 was not found.'
-                    ).arg(unicode(string)), None)
+                    ).arg(identity.decode('utf-8')), None)
```
Comment (PeterSurda) [inline src/namecoin.py:97]: `).arg(identity.decode('utf-8', 'ignore')), None)`

File: src/namecoin.py [hunk 2]
```diff
@@ -130,10 +130,10 @@ def query(self, string):
         return (
             None, "%s <%s>" % (display_name, res)
         ) if valid else (
-            tr._translate(
+            _translate(
                 "MainWindow",
                 'The name %1 has no associated Bitmessage address.'
-            ).arg(unicode(string)), None)
+            ).arg(identity.decode('utf-8')), None)
```
Comment (PeterSurda) [inline src/namecoin.py:136]: also

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/namecoin.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 import sys
 
 import defaults
-import tr  # translate
+from tr import _translate  # translate
```
Comment (g1itch) [inline src/namecoin.py:14]: isort

File: src/namecoin.py (hunk 1 above)
Comment (kdcis) [reply src/namecoin.py]: updated

File: src/namecoin.py [hunk 4]
```diff
@@ -103,16 +103,16 @@ def query(self, string):
...
+            return (_translate(
                 "MainWindow", 'The namecoin query failed (%1)'
-            ).arg(unicode(errmsg)), None)
+            ).arg(errmsg.decode('utf-8')), None)
         except AssertionError:
-            return (tr._translate(
+            return (_translate(
                 "MainWindow", 'Unknown namecoin interface type: %1'
-            ).arg(unicode(self.nmctype)), None)
+            ).arg(self.nmctype.decode('utf-8')), None)
```
[the remark this answers is not in the document]
Comment (kdcis) [reply src/namecoin.py]: updated

File: src/namecoin.py (hunk 2 above)
Comment (kdcis) [reply src/namecoin.py]: updated

File: src/namecoin.py (hunk 3 above)
Comment (kdcis) [reply src/namecoin.py:14]: Not getting it.

Comment (kdcis) [reply src/namecoin.py]: Updated

File: src/namecoin.py [hunk 5]
```diff
@@ -159,19 +158,19 @@ def test(self):
...
                         'Success!  Namecoind version %1 running.').arg(
-                            unicode(versStr)))
+                            versStr.decode('utf-8', 'ignore')))
 
             elif self.nmctype == "nmcontrol":
                 res = self.callRPC("data", ["status"])
                 prefix = "Plugin data running"
                 if ("reply" in res) and res["reply"][:len(prefix)] == prefix:
-                    return ('success', tr._translate("MainWindow", 'Success!  NMControll is up and running.'))
+                    return ('success', _translate("MainWindow", 'Success!  NMControll is up and running.'))
```
Comment (g1itch) [inline src/namecoin.py:170]: line too long

File: src/namecoin.py [hunk 6]
```diff
@@ -235,9 +234,9 @@ def queryHTTP(self, data):
                 result = resp.read()
                 if resp.status != 200:
                     raise Exception("Namecoin returned status %i: %s" % (resp.status, resp.reason))
-            except:
+            except:  # noqa:E722
                 logger.info("HTTP receive error")
-        except:
+        except:  # noqa:E722
```
Comment (PeterSurda) [inline src/namecoin.py:239]: move this `except` up to after `self.con.send(data)` and put the second try/except block outside of the first one.

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/namecoin.py (hunk 5 above)
Comment (g1itch) [reply src/namecoin.py:170]: Also it's better to use double quotes in `_translate()` calls for uniformity:
```python
return (
    'success',
    _translate("MainWindow", "Success!  NMControll is up and running."))
```

File: src/namecoin.py (hunk 6 above)
Comment (kdcis) [reply src/namecoin.py:248]: Updated

File: src/namecoin.py [hunk 7]
```diff
@@ -227,23 +241,28 @@ def queryHTTP(self, data):
...
-            try:
-                resp = self.con.getresponse()
-                result = resp.read()
-                if resp.status != 200:
-                    raise Exception("Namecoin returned status %i: %s" % (resp.status, resp.reason))
-            except:
-                logger.info("HTTP receive error")
-        except:
+        except:  # noqa:E722
             logger.info("HTTP connection error")
```
Comment (PeterSurda) [inline src/namecoin.py:250]: there should be a 
```
return None
```
here

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/namecoin.py (hunk 5 above)
Comment (kdcis) [reply src/namecoin.py]: updated

File: src/namecoin.py (hunk 7 above)
Comment (kdcis) [reply src/namecoin.py:251]: updated

**Review by PeterSurda — APPROVED** [review]

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**